events/filtervalg/README.md

```diff
@@ -0,0 +1,8 @@
+# Filtervalg
+
+En bruker filtrerer visning av informasjon
+
+| Felt | | Type | Beskrivelse |
+| :--- | :--- | :--- | :--- |
+| kategori | påkrevd | string | tekst på filter som brukes |
+| filternavn | påkrevd | string | tekst på filteralternativet som brukes |
```

Comment:
Forskjellen på `kategori` og `filternavn` er ikke intuitiv for meg.

Kunne `filternavn` og `filtreringsverdi` vært et alternativ?